rules/windows/process_creation/win_susp_control_dll_load.yml

```diff
@@ -17,7 +17,7 @@ logsource:
 detection:
     selection:
         ParentImage|endswith: '\System32\control.exe'
-        CommandLine|contains: '\rundll32.exe '
+        Image|endswith: '\rundll32.exe '
```

Comment:
The original rule could be found [here](https://github.com/SigmaHQ/sigma/blob/96eb4609447eba4ce7ad976bbc16af6f0c3d9691/rules/windows/process_creation/win_susp_control_dll_load.yml). Back in January 2019 it had the following logic:

```
detection:
  selection:
    ParentImage: '*\System32\control.exe'
    CommandLine: '*\rundll32.exe *'
  filter:
    CommandLine: '*Shell32.dll*'
```

I am not sure if it should be `Image|endswith: '\rundll32.exe '`. Especially as there is a space in the end. 
IMHO it's better to ask the author for his opinion. @Neo23x0 could you please help us out with the rule?

Reply:
I think this is just a rundll invocation with parameters and the space has no special meaning except separation of command and parameters.